drivers/bluetooth/hci/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) 2024 Silicon Laboratories Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+zephyr_library_sources_ifdef(CONFIG_BT_SIWX917  hci_silabs_siwx917.c)
```

Comment:
GitHub is complaining about missing newline character at the end of this line

Reply:
That's strange, just the file name was changed, I've added the new line.